proto/src/determined/api/v1/task.proto

```diff
@@ -0,0 +1,6 @@
+syntax = "proto3";
```

Comment:
Do we need this?

Reply:
I think so

> The first line of the file specifies that you're using proto3 syntax: if you don't do this the protocol buffer compiler will assume you are using proto2. This must be the first non-empty, non-comment line of the file.

Reply:
I meant the file at all

Reply:
aah I like the idea of having it separate otherwise we'd put in one of the sub tasks and importing from those is a bit more awkward. do you have a suggestion for where to put it?